src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/PosixSignal.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.InteropServices
+{
+    public enum PosixSignal
```

Comment:
`SIGWINCH`, `SIGSTOP`, `SIGCONT`, `SIGTTIN`, `SIGTTOU`, `SIGTSTP`?

Reply:
I can add some more. I started with the ones that were specifically mentioned in https://github.com/dotnet/runtime/issues/50527#issuecomment-825858901

Reply:
Those I listed are commonly used in TUI-style apps. It would definitely be good to have those in the enum so users don't have to hardcode them.

Reply:
I should think that those mentioned in the POSIX standard (https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/signal.h.html) should be the list.

Reply:
> SIGWINCH, SIGCONT, SIGTTIN, SIGTTOU, SIGTSTP?

I've added these. Adding a registration will prevent the process from stopping. `Cancel` doesn't mean anything for these signals.

Reply:
> I should think that those mentioned in the POSIX standard (https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/signal.h.html) should be the list, or if the 2018 list is too modern then, 2008 (https://mirror.math.princeton.edu/pub/oldlinux/download/c950.pdf pg 302)?

@JamesWTruher can you add a literal list of signals you propose get added? Do you have a use for any of these signals?

@jkotas @davidfowl @bartonjs @janvorli do you want all these signals to be added?

Reply:
Signals that are used by the runtime itself such as SIGUSR or SIGBUS are problematic. Attempt to handle them in managed code will lead to crashes or deadlocks. (We would need some special handskake with the runtime itself on how to handle these signals - I am not sure how it would work.)